src/views/accounts/AccountChangeSecret/AccountChangeSecretList.vue

```diff
@@ -114,6 +130,32 @@ export default {
           automation_id: row.id
         }
       })
+    },
+    bulkDisableCallback(selectedRows, reloadTable) {
+      const url = '/api/v1/accounts/change-secret-automations/'
+      const data = selectedRows.map(row => {
+        return { id: row.id, is_active: false }
+      })
+      if (data.length === 0) return
+      this.$axios.patch(url, data).then(() => {
+        reloadTable()
+        this.$message.success(this.$t('common.disableSuccessMsg'))
+      }).catch(error => {
+        this.$message.error(this.$t('common.updateError') + ' ' + error)
+      })
+    },
+    bulkActivateCallback(selectedRows, reloadTable) {
+      const url = '/api/v1/accounts/change-secret-automations/'
+      const data = selectedRows.map(row => {
+        return { id: row.id, is_active: true }
+      })
+      if (data.length === 0) return
+      this.$axios.patch(url, data).then(() => {
+        reloadTable()
+        this.$message.success(this.$t('common.activateSuccessMsg'))
+      }).catch(error => {
+        this.$message.error(this.$t('common.updateError') + ' ' + error)
+      })
     }
   }
 }
```

Comment:
There are no discrepancies found in terms of the code's structure or logic. The code seems to be properly written, following the instructions and guidelines provided. No optimizations were suggested as there was nothing unusual or unexpected about it.

However, some improvements could include using consistent capitalization for function names and variable names (e.g., `bulkActivate` instead of `BulkActivate`). Additionally, separating related UI elements into separate functions would improve modularity and maintainability.